docs/config/README.md

```diff
@@ -9,8 +9,8 @@ mkdir -p ~/.config && touch ~/.config/starship.toml
 All configuration for starship is done in this [TOML](https://github.com/toml-lang/toml) file:
 
 ```toml
-# Don't print a new line at the start of the prompt
-add_newline = false
+# Inserts a newline between shell prompts
```

Comment:
Do you think clarifying that this is  a "blank line" instead of newline will help?

To me, a newline is just a line break, whereas a "blank line" is specifically a whole line with nothing on it--perhaps this will help further clear up some of the confusion? (It doesn't make much sense to insert a blank line in the middle of the prompt).

Reply:
makes sense, updating